servers/physics_2d/godot_joints_2d.cpp

```diff
@@ -155,6 +155,9 @@ bool GodotPinJoint2D::setup(real_t p_step) {
 
 	bias = delta * -(get_bias() == 0 ? space->get_constraint_bias() : get_bias()) * (1.0 / p_step);
 
+	// compute max impulse
```

Comment:
```suggestion
	// Compute max impulse.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).